src/classes/ERR_DeleteOutdated_TEST.cls

```diff
@@ -0,0 +1,54 @@
+/*
+    Copyright (c) 2018 Salesforce.org
+    All rights reserved.
+
+    Redistribution and use in source and binary forms, with or without
+    modification, are permitted provided that the following conditions are met:
+
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of Salesforce.org nor the names of
+      its contributors may be used to endorse or promote products derived
+      from this software without specific prior written permission.
+
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+    "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+/**
+* @author Salesforce.org
+* @group ErrorHandling
+* @description Unit tests for ERR_AsyncErrors
+*/
+@isTest
+public class ERR_DeleteOutdated_TEST {
+    /**
+     * @description Errors outdated should be deleted by a queueable job.
+     */
+    @isTest
+    private static void testOutdatedErrorDeleted() {
+        List<Error__c> errors = UTIL_UnitTestData_TEST.getListOfErrors(2);
+        errors[0].Datetime__c = errors[0].Datetime__c.addDays(-100);
```

Comment:
Might be good to run this on a larger data volume, especially for orgs that have been using HEDA for years.

Reply:
Well, but for the test to be meaningful it'd have to run on a set of over 200. But that may cause the test suite to run longer. I'd like to avoid that, unless we have a good reason. (There is in fact one test in HEDA that runs on a set of over 200 records, and since the tests run in parallel it's the culprit of the tests taking about 10 mins instead of 3).

Reply:
I totally forgot about that test limitation, let's keep it small.